mainnet-1/specs/movement.json

```diff
@@ -0,0 +1,629 @@
+{
+    "proposal": {
+        "title": "Add Specs: Movement",
+        "description": "Adding new specification support for relaying Movement data on Lava",
+        "specs": [
+            {
+                "index": "MOVEMENT",
+                "name": "movement mainnet",
+                "enabled": true,
```

Comment:
please use inheritance to build specs, see example on lavanet repo
https://github.com/lavanet/lava/blob/main/cookbook/specs/movement.json

Reply:
Added APT inheritance